src-java/flowmonitoring-topology/flowmonitoring-messaging/src/main/java/org/openkilda/messaging/info/haflow/UpdateHaSubFlowCommand.java

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.haflow;
+
+import org.openkilda.messaging.command.CommandData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+/**
+ * Represents update ha flow in monitoring command.
```

Comment:
the javadoc could be better. 
You can build javadoc around the answers to: this is a command to a switch? It is related to an HA flow or to an HA sub flow as the class name suggests? Why would anybody want to use this class?

Reply:
So, they are wrong in multiple places. Clearly: HA sub flow and HA flow are totally different things, here the doc is misleading.
Please fix it.